alf/examples/taac_fetch_conf.py

```diff
@@ -22,7 +22,7 @@
 from alf.utils import dist_utils
 
 from alf.examples import sac_conf
-from alf.examples import fetch_conf
+import fetch_conf
```

Comment:
Does this mean that this needs to run under alf/examples/benchmarks/fetch?
Should we do `from alf.examples.benchmarks import fetch_conf` instead?